e2e/steps_file.js

```diff
@@ -111,6 +111,8 @@ module.exports = function () {
       } else {
         this.see(name);
       }
+      dateAndTimeUploaded && this.see(dateAndTimeUploaded);
+      uploadedBy && this.see(uploadedBy);
```

Comment:
minor: I think i'd prefer an if on both but not fussed.

Reply:
As I was saying, I prefer this :) .

Reply:
I think readability of this might be nicer like if statements (easier to read in my opinion)

if (dateAndTimeUploaded) {
    see(dateAndTimeUploaded);
}

Reply:
yeah, if statement is much easier to read, the easier code is to read the better, not everyone who will be reading this code must know js

Reply:
I am surprised TBH. It's not worth discussing about this too much. I will change it to if statements.